docs/life_of_an_event.md

```diff
@@ -0,0 +1,9 @@
+# Life of an Event
```

Comment:
This page would benefit a lot from a flow diagram below the steps. You can create one with markdown - both in GH and Docusaurus I think.